kolibri/core/tasks/scheduler.py

```diff
@@ -60,7 +61,10 @@ def __init__(self, queue=None, connection=None):
             if connection is None:
                 connection = self.queue.storage.engine
         elif connection:
-            self.queue = queue(connection=connection)
```

Comment:
I think it might be better to do this check inside `main.py` can check that the database is working at the initialization of the connection: https://github.com/learningequality/kolibri/blob/release-v0.13.x/kolibri/core/tasks/main.py#L44

Reply:
@rtibbles done